src/components/Sidebar/Sidebar.types.ts

```diff
@@ -26,6 +26,7 @@ export interface ItemPropsExtrinsic {
   children?: ReactElement;
   icon?: ReactElement;
   label: string;
+  labelAsTooltip?: boolean;
```

Comment:
this prop is yucky but felt like the most suitable way to maintain backwards compatibility